cluster/install-olm.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+echo 'Install OLM on cluster'
+./cluster/kubectl.sh apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.10.0/crds.yaml
```

Comment:
can we generate this files with number 0.0.0 to always use the one in develop?

Reply:
this is not generated from our project, those are released OLM manifests. i don't think we should test their development version. maybe once we coverage for one stable, we can extend the matrix

Reply:
sorry I miss that is the operator and not our CR